test/src/test/java/hudson/security/HudsonPrivateSecurityRealmTest.java

```diff
@@ -498,11 +519,69 @@ public void controlCharacterAreNoMoreValid_CustomRegex() throws Exception {
         }
     }
 
+    @Test
+    public void passwordHashWithPBKDF2() {
```

Comment:
there is a Jenkins started and running for all of these tests - but it does not appear that you need one.
if that is the case the test methods can be annotated with [`@WithoutJenkins`](https://javadoc.jenkins.io/component/jenkins-test-harness/org/jvnet/hudson/test/WithoutJenkins.html)

Reply:
updated test method with @WithoutJenkins